google-cloud-clients/google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITSystemTest.java

```diff
@@ -898,39 +909,46 @@ public void queryWatch() throws Exception {
                     @Override
                     public void onEvent(
                         @Nullable QuerySnapshot value, @Nullable FirestoreException error) {
+                      System.out.printf("onEvent(value : %s, error : %s)%n", value, error);
```

Comment:
nit: is this intentionally left here?

Reply:
Yes, I'm wanting to see the events flowing in vs the switch branches below being hit. I haven't yet ruled out if there is some sort of race happening on the events being delivered.